certbot-dns-sakuracloud/docs/index.rst

```diff
@@ -0,0 +1,28 @@
+.. certbot-dns-sakuracloud documentation master file, created by
+   sphinx-quickstart on Wed May 10 18:30:40 2017.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to certbot-dns-sakuracloud's documentation!
+=============================================
```

Comment:
ReStructuredText is a bit picky. You'll need to add more equals characters so that the length of this line matches the one above. This will eliminate the following warning while building the documentation:
```
/certbot/certbot-dns-sakuracloud/docs/index.rst:7: WARNING: Title underline too short.

Welcome to certbot-dns-sakuracloud's documentation!
=============================================
```